Bio/PDB/SASA.py

```diff
@@ -98,7 +98,7 @@ def __init__(self, probe_radius=1.40, n_points=100, radii_dict=None):
                 f"Probe radius must be a positive number: {probe_radius} <= 0"
             )
 
-        self.probe_radius = float(probe_radius)
```

Comment:
Would it matter if the user passed an integer or some other non-float like a NumPy scalar?

Reply:
Right. Let me fix that.